packages/http-client-csharp/emitter/test/Unit/model-type.test.ts

```diff
@@ -426,35 +426,16 @@ op op5(@body body: ExtendsFooArray): ExtendsFooArray;
     assert(extendsFooModel !== undefined);
     assert(extendsFooArrayModel !== undefined);
     // assert the inherited dictionary type is expected
-    deepStrictEqual(
-      {
-        Kind: "any",
-        Encode: undefined,
-      },
-      extendsUnknownModel.AdditionalProperties
-    );
-    deepStrictEqual(
-      {
-        Kind: "string",
-        Encode: undefined,
-      },
-      extendsStringModel.AdditionalProperties
-    );
-    deepStrictEqual(
-      {
-        Kind: "int32",
-        Encode: undefined,
-      },
-      extendsInt32Model.AdditionalProperties
-    );
-    deepStrictEqual(fooModel, extendsFooModel.AdditionalProperties);
-    deepStrictEqual(
-      {
-        Kind: "array",
-        ValueType: fooModel,
-      },
-      extendsFooArrayModel.AdditionalProperties
-    );
+    strictEqual(extendsUnknownModel.AdditionalProperties?.Kind, "any");
```

Comment:
Why does not use `deepStrictEqual` to check the whole object, instead of validate each item of the object?

Reply:
see comment here: https://github.com/Azure/autorest.csharp/pull/4879#discussion_r1660847267